style.css

```diff
@@ -0,0 +1,399 @@
+html {
+  align-items: center;
+  width: 100%;
+  display: flex;
+}
+
+body {
+  background: #e9e8e8b6;
+  position: absolute;
+  width: 100%;
+  border: 1px;
+  margin: 20px;
+  left: 0;
+  top: 0;
+  align-items: center;
+}
+
+header {
+  width: 375px;
+  height: 60px;
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+  padding-left: 0;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+}
+
+.logo {
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.list {
+  display: none;
+}
+
+header p {
+  color: #6070ff;
+  width: 140px;
+  height: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 29px;
+  line-height: 20px;
+  display: flex;
+  text-align: start;
+}
+
+header i {
+  font-size: 2.2rem;
+  margin-right: 50px;
+}
+
+button a {
+  text-decoration: none;
+}
+
+.headline a {
+  text-decoration: none;
+}
+
+.wraper {
+  background-color: white;
+  width: 380px;
+  border-radius: 0 0 0 80px;
+}
```

Comment:
- [ ] Please make sure to use `width: 100%` for sections to make your container take up the whole screen just like the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). You can add the following to your CSS file to remove any default styling
```sh
*,
::before,
::after {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}
```

- [ ] [OPTIONAL] Please 🙏 make your `font size` and `font-weight` of `<p>` tag to look like [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

_**Your design**_  

<img width="359" alt="Screenshot 2023-02-22 at 10 03 04 PM" src="https://user-images.githubusercontent.com/44798044/220704486-7636c159-6adc-4dda-9bee-ea9aa3482c8f.png">

_**Figma design**_

<img width="367" alt="Screenshot 2023-02-22 at 10 02 54 PM" src="https://user-images.githubusercontent.com/44798044/220704534-d51a44c0-7ced-49a1-94e4-faa4a79d16a2.png">